fuzzers/nyx_libxml2_parallel/src/main.rs

```diff
@@ -54,7 +54,7 @@ fn main() {
         let mut feedback = MaxMapFeedback::new(&observer);
         let mut objective = CrashFeedback::new();
         let scheduler = RandScheduler::new();
-        let mut executor = NyxExecutor::new(helper, tuple_list!(observer));
+        let mut executor = NyxExecutorBuilder::new().build(helper, tuple_list!(observer));
```

Comment:
What's our normal naming scheme btw? Should this be `::builder` instead of `new` (or something else?)

Reply:
builder is better i'll change

Reply:
Ok